packs/data/boons-and-curses.db/achaekek-major-boon.json

```
@@ -0,0 +1,38 @@
+{
+    "_id": "eFwCopyvVam6GiCT",
+    "data": {
+        "badge": null,
+        "description": {
+            "value": "<p><strong>Major Boon:</strong> Death lives in your very blade. You gain the @Compendium[pf2e.classfeatures.Sneak Attack]{Sneak Attack} rogue class feature, dealing 3d6 precision damage. if you already have that class feature, you increase your sneak attack damage by 3d6.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 0
+        },
+        "rules": [],
+        "source": {
+            "value": ""
```

```suggestion
            "value": "Pathfinder Loast Omens: Gods and Magic"
```
Make this change on every item in this MR